easybuild/easyconfigs/b/BLAST/BLAST-2.2.26-Linux_x86_64.eb

```diff
@@ -16,7 +16,7 @@ description = """Basic Local Alignment Search Tool, or BLAST, is an algorithm
 
 toolchain = SYSTEM
 
-source_urls = ['https://ftp.ncbi.nlm.nih.gov/blast//executables/legacy/%(version)s/']
+source_urls = ['https://ftp.ncbi.nlm.nih.gov/blast/executables/legacy.NOTSUPPORTED/%(version)s/']
 sources = ['%(namelower)s-%(version)s-x64-linux.tar.gz']
```

Comment:
@zemu-unile Please add a SHA256 checksum, `eb --inject-checksums` is very handy for this.